tests/unit/app/validations/modify_grow_test.py

```diff
@@ -1,10 +1,31 @@
 from app.validation.modify_grow import validate_modify_grow
-from unittest.mock import MagicMock
+from unittest.mock import MagicMock, Mock
 import pytest
 
+
 def test_modify_grow():
     AppConfig = MagicMock()
-    assert validate_modify_grow(AppConfig, {"grow_phases": 1, "grow_id": 5, "end_date": 3, "recipe_name": "OGK"}) == (True, "")
+    mock = Mock()
+    assert validate_modify_grow(
+        AppConfig, {"grow_phases": 1, "grow_id": 5, "end_date": 3, "recipe_name": "OGK"}
+    ) == (True, "")
+    mock.validate_modify_grow(
+        AppConfig, {"grow_phases": 1, "grow_id": 5, "end_date": 3, "recipe_name": "OGK"}
+    )
```

Comment:
Why is this line needed? I'm also fine with leaving out the argument check on lines 15-17, as long as we verify that the return value is correct.